python/desc/skycatalogs/utils/sed_tools.py

```diff
@@ -154,7 +155,7 @@ class MilkyWayExtinction:
     '''
```

Comment:
I sent an alternative implementation for `MilkyWayExtinction` that doesn't use any info from `sed_factory` and should apply more generally to SEDs from any kind of object.